integration/test/Testlib/RunServices.hs

```diff
@@ -57,6 +77,36 @@ main = do
       $ do
         _modifyEnv <-
           traverseConcurrentlyCodensity
-            (\r -> startDynamicBackend r mempty)
+            ( \r ->
+                if opts.withManualTestingOverrides
+                  then startDynamicBackend r manualTestingOverrides
+                  else startDynamicBackend r mempty
+            )
             [backendA, backendB]
         liftIO run
+
+manualTestingOverrides :: ServiceOverrides
+manualTestingOverrides =
+  let smtpEndpoint = object ["host" .= "localhost", "port" .= (2500 :: Int)]
+      authSettings =
+        object
+          [ "userTokenTimeout" .= (4838400 :: Int),
```

Comment:
I'm usually too lazy to calculate and would write a `let` or `where` binding for such numbers. E.g. `fiftysixDays = 4838400`.

But, that's probably a matter of taste :smile:

Reply:
I copy pasted these from default values of the helm charts and I have no clue what they correspond to, I'd wager noone would care about this until they testing something specific at which point they'd probably just change the values temporarily.